.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: AntennaPod Forum
+    url: https://forum.antennapod.org/
+    about: Reduce developer's workload by asking other users.
```

Comment:
Turn it positive, and make it user-centric:
"Get quick help with and give feedback on AntennaPod"